frontend/components/TableContainer/DataTable/StatusCell/StatusCell.tsx

```diff
@@ -12,13 +17,31 @@ const generateClassTag = (rawValue: string): string => {
   return rawValue.replace(" ", "-").toLowerCase();
 };
 
-const StatusCell = ({ value }: IStatusCellProps): JSX.Element => {
+const StatusCell = ({ value, tooltip }: IStatusCellProps): JSX.Element => {
   const statusClassName = classnames(
     "data-table__status",
     `data-table__status--${generateClassTag(value)}`
   );
-
-  return <span className={statusClassName}>{value}</span>;
+  const cellContent = tooltip ? (
+    <>
+      <span data-tip data-for={tooltip.id}>
```

Comment:
data-for={`tooltip-id-${tooltip.id}`} ? Idk, just brainstorming how we can use an optional id, maybe it is better to always pass an id

Reply:
I think it's worth changing it to `status-${tooltip.id}` so it's more unique. Otherwise, there's a chance another element may have the same ID, which could cause a bug.